uber/templates/jobs/index.html

```diff
@@ -67,20 +124,72 @@
     </tr>
 {% endfor %}
 </table>
+-->
+{% endblock %}
+{% block page_scripts %}
+<script type="text/javascript" src="../static/js/moment.js"></script> 
 
-<div style="border:1px">
-    <h3>Teardown Shifts</h3>
-    <a href="form?id=None&location={{ location }}&type={{ c.TEARDOWN }}">Add a teardown shift</a>
-    <ul>
-    {% for job in teardown %}
-        <li>
-            [<b>{{ job.start_time|datetime:"%I %p %a" }}</b> for an expected ~{{ job.duration }} hours]
-            <a href="form?id={{ job.id }}">{{ job.name }}</a>:
-            {{ job.shifts|length }}/{{ job.slots }} Staffers <a href="staffers_by_job?id={{ job.id }}">Signed Up</a>
-        </li>
-    {% endfor %}
-    </ul>
-    <br/>
-</div>
 
+<script type="text/javascript" src="../static/js/fullcalendar.min.js"></script> 
+<script type="text/javascript"> 
+
+
+	$(document).ready(function() {
+
+		var eventList =  new Array();
+		    {% for job in jobs %}
+                    eventList.push({
+                                        title : "{{job.name}} ({{job.shifts | length }}/{{job.slots}})",
+                                        start : "{{ job.start_time|datetime:"%Y-%m-%dT%H:%M:%S" }}",
+                                        // duration {{job.duration}}
+                                        end: "{{job.start_time|shift_end:job.duration}}",
+                                        url : "form?id={{ job.id }}",
+                                        {% if job.type == c.SETUP or job.type == c.TEARDOWN %}
+                                        backgroundColor: '#239875'
+                                        {% else %}
+                                        backgroundColor:'#305fc9'
+                                        {% endif %}
+                                });
+    		    {% endfor %}
+
+		$('#shift_cal').fullCalendar({
+			header: {
+				left: 'prev,next today',
```

Comment:
Silly question, but is there any way to have the buttons that say "Today," "Week," "Day," and "List" capitalized?